src/types.jl

```diff
@@ -172,22 +172,22 @@ end
 # Compat only
 function Base.getproperty(e::Events, field::Symbol)
     if field === :mousebuttons
-        error(
-            "`events.mousebuttons` is deprecated. Use `events.mousebutton` to " *
-            "react to `MouseButtonEvent`s instead and ``."
-        )
+        error("""
```

Comment:
I think this way of writing the error strings needs backslashes to avoid linebreaks, which is a rather new feature iirc.

Reply:
Ah... But do we want to avoid them? I have the rule of thumb, that if it's nice in the code to have line breaks, it's probably nice in the print out as well.

Reply:
I think the line breaks here were just to stay in an 80 characters/line limit or something similar. 

Don't log messages also handle line breaks themselves?

Reply:
I made the linebreak a bit nicer.. I think it's good to break the line like that for the error message

Reply:
nevermind :D I looked at the actual error some more, and it does look better in one line.
If someone doesn't like such long lines in the code, my general philosophy has been to enable word wrap in the editor and just keep semi long lines in code.